nrrd/tests/test_writing.py

```diff
@@ -228,7 +228,7 @@ def test_write_detached_ascii(self):
         data, header = nrrd.read(output_filename)
         self.assertEqual(self.expected_data, data.tostring(order='F'))
         self.assertEqual(header['encoding'], 'txt')
-        self.assertEqual(header['data file'], output_data_filename)
+        self.assertEqual(header['data file'], os.path.basename(output_data_filename))
```

Comment:
`output_data_filename` is equal to joined path of temp dir and the filename. We then do the basename to get the filename. 

Easier if we just replace `output_data_filename` with the relative filename.

Repeat for above case too.